src/modules/book_module.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../classes/book'
+
+module BookModule
+  def find_books
+    puts 'Books: '
+    if @books.empty?
+      puts "\nNo  books added yet. Please add some  . . . "
+    else
+      @books.each_with_index do |book, index|
+        puts "[Book #{index + 1}]. Publisher : #{book.publisher},+
+        Publish Date : #{book.publish_date}, Cover State : #{book.cover_state}"
```

Comment:
- [OPTIONAL] When listing the Books, I would advise you to remove the `+` sign at the end of the publisher's name so as not to confuse users about its relevance. 👍